Build/scripts/build.ps1

```diff
@@ -0,0 +1,93 @@
+#-------------------------------------------------------------------------------------------------------
+# Copyright (C) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+#-------------------------------------------------------------------------------------------------------
+
+# Build Script
```

Comment:
Note to self (and reviewers): I think it would be better to rename this file to something like "run_build.ps1" or "do_build.ps1" to reserve the name of "build.ps1" for a script which runs the entire build.

(After discussing offline with @Cellule it seems like it is more and more likely that a single script is going to be more or less where the MicroBuild definition goes - we're already converging to an entirely powershell-script-based build.)